bom/runtime/pom.xml

```diff
@@ -446,7 +446,7 @@
     <dependency>
       <groupId>com.neuronrobotics</groupId>
       <artifactId>nrjavaserial</artifactId>
-      <version>3.14.0</version>
+      <version>3.15.0</version>
```

Comment:
That's just for testing (`runtime` POM is used in itests and for the Bnd based demo applications).